packages/shared/src/components/post/SquadPostWidgets.tsx

```diff
@@ -20,7 +20,7 @@ import { FooterLinks } from '../footer';
 
 const SquadCard = ({ squadSource }: { squadSource: Squad }) => {
   const { isFetched } = useAuthContext();
-  const { id: squadId, handle } = squadSource;
+  const { id: squadId, handle } = squadSource || {};
```

Comment:
These are so annoying :triggered:

Reply:
should have been handled in parent component, but would require too much changes

Reply:
also strict type checking when 😆